opt/prepare-aiidalab.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+# Debugging.
+set -x
+
+# Environment.
+export SHELL=/bin/bash
+
+
+# Setup CP2K code.
+computer_name=localhost
+code_name=cp2k
+verdi code show ${code_name}@${computer_name} || verdi code setup \
+    --non-interactive                                             \
+    --label ${code_name}                                          \
+    --description "cp2k on this computer"                         \
```

Comment:
Please put something that is more useful for provenance - imagine someone finds a calculation done with this code in 10 years. What do they want to know?

E.g. "cp2k v... in AiiDA lab container"

Reply:
done